jest.config.mjs

```diff
@@ -1,13 +1,21 @@
 // jest.config.mjs
 export default {
   testEnvironment: "jest-environment-jsdom",
-  transform: {
+  /* transform: {
     "^.+\\.jsx?$": "babel-jest",
   },
-  moduleNameMapper: {
+ moduleNameMapper: {
     "^@/(.*)$": "<rootDir>/src/$1",
-  },
+  },*/
```

Comment:
Could you go ahead and remove the commented out lines.

Reply:
I have removed the comment as requested.